evidence-display/scripts/deploy.sh

```
@@ -0,0 +1,110 @@
+      echo "Invalid option: -${OPTARG}."
+      usage
+      exit 2
+      ;;
+  esac
+done
+
+SCRIPTS_DIR=scripts
+BUILD_DIR=build
+COMPACTED_FILE_NAME="xchain-realitio-evidence-display.tar.gz"
+COMPACTED_FILE_PATH="${SCRIPTS_DIR}/${COMPACTED_FILE_NAME}"
+REMOTE_DIRECTORY='~/cross-chain-realitio'
```

Shellcheck (0.10.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)